bigtop-packages/src/charm/mahout/layer-mahout/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+## Overview
+
+
+The Apache Mahout™ project's goal is to build an environment for quickly creating
+scalable performant machine learning applications.
+
+Apache Mahout software provides three major features:
+ * A simple and extensible programming environment and framework for building scalable algorithms
+ * A wide variety of premade algorithms for Scala + Apache Spark, H2O, Apache Flink
+ * Samsara, a vector math experimentation environment with R-like syntax which works at scale
+
+
+## Usage
+
+This charm is intended to be added to an existing cluster
+for example a Hadoop one. A Hadoop cluster can be deployed via
+one of the [apache bigtop bundles](https://jujucharms.com/u/bigdata-charmers/#bundles).
+For example:
+
+    juju deploy hadoop-processing
+
+> Note: With Juju versions < 2.0, you will need to use [juju-deployer][] to
+deploy the bundle.
+
+This will deploy the Apache Bigtop Hadoop platform with a workload node
+preconfigured to work with the cluster.
+
+You can then add Mahout by relating it to the client:
+
```

Comment:
Is it worth mentioning it can be related to any charm which uses the hadoop-client layer?